spec/factories.rb

```diff
@@ -0,0 +1,26 @@
+FactoryGirl.define do
+  factory :challenge do
+    title :test
+    description :test
+    input :a
+    output :b
+    diff :c
+    user
+  end
+
+  factory :entry do
+    script :a
+    created_at Time.now.utc
```

Comment:
not sure when `Time.current` was introduced in rails but it does this exact thing. totally fine to leave for now. this is a good find tho, could cause some test inconsistencies :)